android/src/main/kotlin/com/jhomlala/better_player/BitmapSingleton.kt

```diff
@@ -0,0 +1,27 @@
+package com.jhomlala.better_player
+
+import android.graphics.Bitmap
+
+// Storing a large bitmap instead of passing it through an intent.
+class BitmapSingleton private constructor() {
+    private var bitmap: Bitmap? = null
+
+    fun setBitmap(newBitmap: Bitmap) {
+        bitmap = newBitmap
+    }
+
+    fun getBitmap(): Bitmap? {
```

Comment:
Sorry.
I cannot understand why we need this class.
(It seems, `getBitmap` is not used.)

Would you describe more why we need this class?

Reply:
> Since the bitmap has already been loaded, I will store it in a singleton class and use it in the BetterPlayerNotificationService instead of reloading the image once again. I'm using a singleton instead of an intent because at times, a large bitmap could lead to a TransactionTooLargeException

Reply:
Thanks. 👍 
I understand we will use it from application code.